src/main.rs

```diff
@@ -365,6 +397,57 @@ fn get_help_menu() -> ArgMatches<'static> {
         .arg(Arg::with_name("open")
             .long("open")
             .help("Open the report in a browser once it is complete."))
+        .arg(Arg::with_name("threads")
+            .short("t")
+            .long("threads")
+            .help("Number of threads to use.")
+            .takes_value(true))
+        .arg(Arg::with_name("downloads-folder")
+            .short("d")
+            .long("downloads-folder")
+            .help("Folder where the downloads are stored.")
+            .takes_value(true))
+        .arg(Arg::with_name("dist-folder")
+            .short("D")
+            .long("dist-folder")
+            .help("Distribution folder.")
```

Comment:
Maybe a description such as *"Folder where distribution files will be extracted"* would be better.